docs: add "unofficial Primeleague API" keyword (#28)
random-rip/primebot_backend: README.md

</p>
<h4 align="center">
The unofficial <i>Strauss Prime League</i> Bot for sending push notifications via Discord and

> Telegram. The unofficial <i>Strauss Prime League</i> API.

Reviewer: ```suggestion
Telegram and API Provider of <i>Strauss Prime League</i> data.
```
